deployment/desktop/hosting-metadata.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Hosting metadata for software applications 
+subsection: desktop
+description: >
+  How to host your GUI desktop application's metadata for yum/DNF or Flatpak in self- or Github-hosted repositories
+---
+
+# Hosting metadata for yum/DNF or Flatpak
+
+Metadata for distribution tools and GNOME Software must be refreshed
+periodically, such as when new versions of [packaged
+applications](desktop-software.html) are released for consumption by
```

Comment:
Where should this link point? I'd say no such path exists. Please use full paths like, f.e. `/tech/languages/csharp/csharp-ide.html`.

Reply:
Invariably I missed a couple links while rewriting this heavily. It was a very substantial amount of change.

Reply:
Sure, don't worry, we just need those fixed. That's what reviews are for.